website/pages/docs/guides/dependencies.mdx

```diff
@@ -13,11 +13,11 @@ To add a dependency, simply declare it in your `Scarb.toml`.
 ## Adding a dependency
 
 If your `Scarb.toml` doesn't already have a `[dependencies]{:toml}` section, add it, then list the package name and the URL to its Git repository.
-This example adds a dependency on the [`quaireaux`](https://github.com/keep-starknet-strange/quaireaux) package:
+This example adds a dependency on the [`quaireaux_math`](https://github.com/keep-starknet-strange/quaireaux) package:
```

Comment:
Please, describe here that `quaureaix` is a collection of multiple packages and we will use math as an example in this guide